core/geometry/src/polyface/PolyfaceQuery.ts

```diff
@@ -618,12 +699,30 @@ export class PolyfaceQuery {
     }
   }
   /**
-   * Invoke the callback on each manifold edge whose adjacent facet normals form vectorToEye dot products with opposite sign.
```

Comment:
Moved "createIndexedEdges" from line 1609 here (no changes to doc or body) so the function is defined in the code before it's called in other functions. Maybe I should not have moved it in this PR because I wrote list of a few functions that should be moved and I will move them in a separate PR to make review easier.